src/site/content/en/blog/content-visibility/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 'CSS Content Visibility: the most impactful performance boost CSS can give you'
+subhead: TBD
+authors:
+  - vladimirlevin
+  - una
+date: 2020-08-05
+hero: hero.jpg
+alt: Stylized photo of a half-hidden person.
+description: TBD
+tags:
+  - blog
+  - css
+---
+
+The [`content-visibility`](https://drafts.csswg.org/css-contain/#propdef-content-visibility) property might be one of the most impactful new CSS properties for improving page load performance. `content-visibility` enables the user agent to skip an element's rendering work, including layout and painting, until it is needed. If a large portion of your content is off-screen, leveraging the `content-visibility` property allows the browser to skip the rendering working for this content, making the initial user load much faster. Additionally, it allows for faster interactions with the on-screen content. Pretty neat.
+
+`content-visibility` relies on primitives within the [the CSS Containment Spec](http://drafts.csswg.org/css-contain/). So, what exactly is this spec? Let's dive in.
+
+## CSS Containment {: #containment }
+
+ -- Caniuse Image Here: caniuse.com/#feat=css-containment --
+
+The key and overarching goal of CSS containment is to enable rendering performance improvements of web content by providing *predictable isolation of a DOM subtree* from the rest of the page.
+
+Basically a developer can tell a browser what parts of the page are encapsulated as a set of content, allowing the browsers to reason about the content without needing to consider state outside of the subtree. Knowing which bits of content (subtrees) contain isolated content means the browser can make optimization decisions for page rendering.
+
+There are four types of [CSS containment](https://developers.google.com/web/updates/2016/06/css-containment), each a potential value for the `contain` CSS property. You can also combine them individually or together in a space-separated list of values:
+
+- `size`: Size containment ensures that the containing box can be laid out without needing to examine its children. The parent sets the geometric size independent of its contents.
+- `layout`: Layout containment means that the subtree does not affect the external layout of other boxes on the page.
+- `style`: Style containment ensures that properties which can have effects on more than just its descendants don't escape the element (e.g. counters).
+- `paint`: Paint containment ensures that the descendants of the containing box don't display outside its bounds. Nothing can visibly overflow the element, and if an element is off-screen or otherwise not visible, its descendants will also not be visible.
+
+## `content-visibility`
```

Comment:
```suggestion
## Skipping rendering work with `content-visibility`
```
@una I updated each of these sections to use the format "X with Y" where X is a generic description of the task and Y is the property name. I feel that this is easier to scan and is less intimidating for people who are learning about these concepts for the first time. WDYT? Feel free to rephrase any of them.

Reply:
I like it!